src/components/Nursery/index.tsx

```diff
@@ -108,13 +107,8 @@ export default function NurseryDetails(): JSX.Element {
           />
         </Grid>
         {timeZoneFeatureEnabled && (
-          <Grid item xs={gridSize()}>
-            <TimeZoneSelector
-              selectedTimeZone={nursery?.timeZone || defaultTimeZone.id}
-              onTimeZoneSelected={() => true}
-              disabled={true}
-              label={strings.TIME_ZONE}
-            />
+          <Grid item xs={gridSize()} marginTop={isMobile ? 3 : 0}>
+            <TextField label={strings.TIME_ZONE} id='timezone' type='text' value={tz.longName} display={true} />
```

Comment:
textfield seems right to use here, re: the checkbox, we will be showing that info the tooltip i suppose - or add other text.